lib/IRGen/IRGenSIL.cpp

```diff
@@ -928,7 +928,23 @@ class IRGenSILFunction :
       }
     }
   }
-  
+
+  // SWIFT_ENABLE_TENSORFLOW
+  // Returns the LLVM function with `funcName`. It must exist.
+  llvm::Function *findFunction(StringRef funcName, SILModule &silModule) {
+    LLVM_DEBUG(llvm::dbgs() << "IRGen for " << funcName << "().\n");
+    auto silFn = silModule.findFunction(funcName, SILLinkage::PublicExternal);
+    assert(silFn);
+    llvm::Function *fn = IGM.getAddrOfSILFunction(silFn, NotForDefinition);
+    assert(fn);
+    return fn;
+  }
+
+  void checkOk(llvm::Value *status) {
```

Comment:
CheckOk is a really generic name, but it’s tied to TFC. After prefixes for runtime functions are added, this function should be renamed accordingly to prevent confusion with the rest of the IRGen code base.

Reply:
Added the TFC prefix.